build.sh

```
@@ -0,0 +1,75 @@
+VERSION_TAG=$2
+
+print_header() {
+    echo "======================================================"
+    echo "🚀 ${1}"
+    echo "======================================================"
+}
+
+# Load common environment variables first
+if [ -f .env ]; then
+    echo "Loading common configuration from .env file..."
+    export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)